keps/sig-release/2572-release-cadence/kep.yaml

```diff
@@ -0,0 +1,38 @@
+title: Defining the Kubernetes Release Cadence
+kep-number: 2572
+authors:
+  - "@justaugustus"
```

Comment:
Please add all the authors (i see many commits from folks)

Reply:
@dims working on that now

Reply:
thanks!